docs/x509/verification.rst

```diff
@@ -21,3 +21,44 @@ chain building, etc.
 
     :param certs: A list of one or more :class:`~cryptography.x509.Certificate`
         instances.
+
+.. class:: Subject
+
+    .. versionadded:: 42.0.0
+
+    Type alias: A union of all subject types supported:
+    :class:`cryptography.x509.general_name.DNSName`,
+    :class:`cryptography.x509.general_name.IPAddress`.
+
+
+.. class:: PolicyBuilder
+
+    .. versionadded:: 42.0.0
+
+    A PolicyBuilder provides a builder-style interface for constructing a
+    Policy.
+
+    .. classmethod:: webpki
+
+        Creates a new :class:`PolicyBuilder` with defaults for the Web PKI,
+        i.e. client verification of web server TLS certificates.
+
+    .. method:: subject(new_subject)
+
+        Sets the policy's subject name.
+
+        :param new_subject: The :class:`Subject` to use in the policy
+
+        :returns: A new instance of :class:`PolicyBuilder`
+
```

Comment:
My recollection is that what we discussed was having `build_server_policy()` and `build_client_policy()` and subject would be passed to `build_server_policy()`.

Reply:
https://github.com/pyca/cryptography/pull/9601/commits/9aec060009b03117bafa21b6d94bceed0beffedf